IOMC/EventVertexGenerators/python/VtxSmearedRealistic_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+# Load BetafuncEvtVtxGenerator and read parameters from GT (SimBeamSpotObjectRcd)
+from IOMC.EventVertexGenerators.BetafuncEvtVtxGenerator_cfi import BetafuncEvtVtxGenerator
+VtxSmeared = BetafuncEvtVtxGenerator.clone(
+    src = "generator:unsmeared",
```

Comment:
@francescobrivio , I may not understand things ut why "generator:unsmeared" if parameters are taken from DB?

Reply:
This comes from the fact that both generators inherit from `BaseEvtVtxGenerator` which needs the `HepUnsmearedMCEvt` source token:
https://github.com/cms-sw/cmssw/blob/29909891e150c9781f4ade2a6f7b5beb0bd67a6e/IOMC/EventVertexGenerators/src/BaseEvtVtxGenerator.cc#L38
and
https://github.com/cms-sw/cmssw/blob/29909891e150c9781f4ade2a6f7b5beb0bd67a6e/IOMC/EventVertexGenerators/src/BaseEvtVtxGenerator.cc#L50

So far this was defined in `VtxSmearedCommon`:
https://github.com/cms-sw/cmssw/blob/3b943c0dbbdf4494cd66064a5a147301f38af295/IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py#L12-L15
now we want an identical setup but reading from DB.